mac.sh

```diff
@@ -197,7 +197,7 @@ cask_install_or_upgrade 'chromedriver'
 
 # Install applications
 cask_install_or_upgrade 'google-chrome'
-install_ruby_2_4_5
+install_ruby_2_5_8_
```

Comment:
is this an extra underscore at the end? otherwise lgtm!

Reply:
thanks! removed.